syntax = "proto3";

package api.filter;

service FilterService {
  rpc ApplyFilters(FilterRequest) returns (FilterResponse);
}

message FilterRequest {
  string image_id = 1;
  map<string, float> filters = 2;
}

message FilterResponse {
  string filtered_url = 1;
}
